### PR TITLE
PRO-410 Wallet Design Revamp

### PR DESCRIPTION
**Pull request by hemadri-tank-bacancy**
* Points

For Wallet Tab
  * When tapping on All (it combines all the USD value of tokens + stables and updates the Portfolio USD number)
  * When tapping on Tokens (this updates the Portfolio USD number to Tokens total value only)
  * When tapping on Stables (this updates the Portfolio USD number to stables total value only)
  * When scrolling down, it will make the search box appear.
  * Add information icon pop out.(Portfolio riskiness)
  * Add chain filter, when chain is selected, the amount is updated for that chain automatically.

For Apps Tab
  * Shows positions per protocol of Investments and values.
  * Total USD value of all investments.
  * Fold out for specific position per app.

* Updated UI design in Home screen in Apps section

https://user-images.githubusercontent.com/78720269/218708525-9a2c1292-ccee-4ecc-965e-f2f3a234e68b.mp4

### CODE REVIEW

**Comment by hemadri-tank-bacancy** [comment]
* Updated some conditions and reduce space in balance content
Simulator Screen Shot - iPhone 13 Pro Max - 2023-02-15 at 15 48 00

**Comment by hemadri-tank-bacancy** [comment]
* Disappears placeholder in search bar on focus.


https://user-images.githubusercontent.com/78720269/219300696-0d64832d-1ed6-46fd-8855-508ca68221ec.mp4

File: src/components/Wallet/WalletSelection.tsx [hunk 1]
```diff
@@ -107,7 +101,7 @@ export default function () {
       <Spacing h={10} />
 
       <FlatList
-        key={items[tabIndex].key}
+        key={selectedTabNm + '__'}
         data={listOfAssets.slice(0, 5)}
         renderItem={({ item, index }) => renderItem(item, index)}
         keyExtractor={(item) => item.address + '__' + item.chain}
```
Comment (IAmKio) [inline src/components/Wallet/WalletSelection.tsx:107]: Whilst it's not wrong, just a notice to use string literals in future.

File: src/screens/Assets/wallet/WalletTabScrollContent.tsx [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+  const renderItem = (token: any) => {
+    return (
+      <TokenListItem
+        chain={token.chain}
+        address={token.address}
+        symbol={token.symbol}
+        name={token.name}
+        iconUrl={token.iconUrl}
+        balance={wrapBigNumberOrNil(token.balance?.balance)}
+        onPress={async () => {
```
Comment (IAmKio) [inline src/screens/Assets/wallet/WalletTabScrollContent.tsx:171]: Do you need the `async` here?

**Review by IAmKio — APPROVED** [review]
Just one small note thing but it's not a show stopper

**Pull request merged**